code/game/gamemodes/miniantags/guardian/guardian.dm

```diff
@@ -701,6 +702,23 @@
 		to_chat(user, "[failure_message]")
 		used = FALSE
 
+/obj/item/weapon/guardiancreator/afterattack(var/obj/item/I, var/mob/user, var/proximity)
+	..()
+	if(!proximity)
+		return
+	if(!refundable)
+		return
+	if(istype(I))
+		if(I.hidden_uplink && I.hidden_uplink.active)
+			if(used == TRUE)
+				to_chat(user, "[used_message]")
+				return
+			var/datum/uplink_item/U = /datum/uplink_item/dangerous/guardian
+			I.hidden_uplink.uses += initial(U.cost)
+			to_chat(user, "<span class='notice'>You send [src] back, getting a refund for its TC.</span>")
+			qdel(U)
```

Comment:
I was creating it - then to address Tiger's comments, I stopped creating it.
I just forgot to remove the qdel. And yes, I tested it before that and it worked fine - I forgot to remove the qdel with Tiger's changes.